e2e/protractor.conf.js

```diff
@@ -2,14 +2,23 @@
 // https://github.com/angular/protractor/blob/master/lib/config.ts
 
 const { SpecReporter } = require('jasmine-spec-reporter');
+var fs = require('fs');
```

Comment:
any reason you went for a var over const?

Reply:
nope, just was looking into how to read files in typescript. will change it.